functional/ek-cert-use-ek_handle-custom-ca_certs/main.fmf

```diff
@@ -0,0 +1,34 @@
+summary:  Test of keylime checking of endorsement key by manually generate ek and verify against CA certs
+description: |
+    Running all services on localhost.
+    Uses certificates generated by keylime.
```

Comment:
I guess this applies to TLS certs, could you please add it (or maybe even remove, it is not important for test scenario).

Reply:
Yeah, I removed it, it's not necessary to describe here.